javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,128 @@
+let userName;
+
+const toDoList = [];
+
+const userNameIntro = ['Hello', 'my', 'name', 'is'];
+const userNameRequest = ['What', 'is', 'my', 'name'];
+const userAddToDos = ['Add', 'to', 'my', 'todo'];
+const userRemovedFromToDos = ['Remove', 'from', 'my', 'todo'];
+const userRequestToDo = ['What', 'is', 'on', 'my', 'todo'];
+const userRequestDay = ['What', 'day', 'is', 'it', 'today?'];
+const userRequestCalculation = ['What', 'is'];
+const userRequestSetTimer = ['Set', 'a', 'timer', 'for', 'minutes'];
+const userRequestMood = ['How', 'is', 'your', 'mood', 'today?'];
+
+const findDifferenceBetweenReqAndRes = (request, possibleReq) => {
+  return request.filter((item) => !possibleReq.includes(item)).join(' ');
+};
+
+const checkForSimilarity = (userRequest, possibleRequest) => {
```

Comment:
the similarity or the equality, since you are using `every`?

Reply:
I'm not sure that I can tell you the difference is this situation 😐 It's looking like equality

Reply:
It is equality 🙂  This comment is more from the perspective of giving variables/functions good names